src/uk/gov/defra/ffc/Pact.groovy

```diff
@@ -15,10 +15,23 @@ class Pact implements Serializable {
         for (pact in pacts) {
           def provider = pact.name.substring("$repoName-".length(), pact.name.indexOf(".json"))
           ctx.echo "Publishing ${pact.name} to broker"
-          
-          def script = "docker run --rm -w \$(pwd) -v \$(pwd):\$(pwd) -e PACT_DISABLE_SSL_VERIFICATION=false -e PACT_BROKER_BASE_URL=$ctx.PACT_BROKER_URL -e PACT_BROKER_USERNAME=$ctx.pactUsername -e PACT_BROKER_PASSWORD=$ctx.pactPassword pactfoundation/pact-cli:latest broker publish --consumer-app-version $version+$commitSha $pact --tag main"
+
+          def script = '''
+            docker run --rm -w \$(pwd) -v \$(pwd):\$(pwd) -e PACT_DISABLE_SSL_VERIFICATION=false \
+            -e PACT_BROKER_BASE_URL=\$PACT_BROKER_URL -e PACT_BROKER_USERNAME=$pactUsername \
+            -e PACT_BROKER_PASSWORD=$pactPassword pactfoundation/pact-cli:latest \
+            broker publish --consumer-app-version \
+            ''' + version + '''+''' + commitSha + ''' ''' + pact + ''' --tag main
+            '''
+
             ctx.gitStatusWrapper(credentialsId: 'github-token', sha: Utils.getCommitSha(ctx), repo: Utils.getRepoName(ctx), gitHubContext: GitHubStatus.PactBrokerPublish.Context, description: GitHubStatus.PactBrokerPublish.Description) {
-            ctx.sh(returnStatus: true, script: script)
+              def output = ctx.sh(returnStatus: true, script: script)
```

Comment:
Capture the output and error if it isn't a zero (i.e. is was successful), error the step. Without this, the publishing of pacts can fail without the step reporting it correctly.